system-int-test/test/cmr/system_int_test/search/variable/collection_measurement_search_test.clj

```diff
@@ -0,0 +1,85 @@
+(ns cmr.system-int-test.search.variable.collection-measurement-search-test
+  "Tests searching for collections by associated measurements"
+  (:require
+   [clojure.test :refer :all]
+   [cmr.common.util :refer [are3]]
+   [cmr.mock-echo.client.echo-util :as e]
+   [cmr.system-int-test.data2.core :as d]
+   [cmr.system-int-test.data2.umm-spec-collection :as data-umm-c]
+   [cmr.system-int-test.system :as s]
+   [cmr.system-int-test.utils.index-util :as index]
+   [cmr.system-int-test.utils.ingest-util :as ingest]
+   [cmr.system-int-test.utils.search-util :as search]
+   [cmr.system-int-test.utils.variable-util :as vu]))
+
+(use-fixtures :each
+              (join-fixtures
+               [(ingest/reset-fixture {"provguid1" "PROV1"})
+                vu/grant-all-variable-fixture]))
+
+(deftest collection-measurement-search-test
+
+  (let [[coll1 coll2 coll3 coll4] (for [n (range 1 5)]
+                                    (d/ingest-umm-spec-collection
+                                     "PROV1"
+                                     (data-umm-c/collection
+                                      {:ShortName (str "S" n)
```

Comment:
You don't need to change it, but so you know, you can pass n into `(data-umm-c/collection)` to create the unique collections. If you don't need to specify the short name/version/entry title.